Contents/mods/Hydrocraft/media/scripts/Lab.txt

```diff
@@ -58,6 +62,7 @@ item HCGlasswaterbottle
 	ReplaceOnUseOn		=	WaterSource-HCGlasswaterbottle,
 	Icon			=	HCGlasswaterbottle,
 	RainFactor 		= 	1,
+		DisplayCategory          =   FoodB,
```

Comment:
craftchem. although you can technically drink from it, I think people would want to keep this with their chem stuff sinces it is used in some chem crafting